### ISSUE TITLE
countries causing errors in 0.8.0

### ISSUE THREAD

**Issue by tomkralidis** [barrust/pyspellchecker#168]
Hi: thanks much for this valuable tool!  Moving to 0.8.0 causes issues when spellchecking text with country names.

Example using 0.7.3:

```python
>>> from spellchecker import SpellChecker
>>> s = SpellChecker()
>>> s.unknown(['Austria'])
set()
```

Example using 0.8.0:
```python
>>> from spellchecker import SpellChecker
>>> s = SpellChecker()
>>> s.unknown(['Austria'])
{'austria'}
```

I am guessing #165 resulted in these changes, but not sure exactly where or why.

Is this expected behaviour?  Thanks in advance.

**Comment by barrust** [comment]
Yes, it looks like the country names did not make it into the new dictionaries. You can add them locally or we could add them into the `en_include.txt` file to ensure that they get added in future releases of the dictionaries.

**Comment by tomkralidis** [comment]
Thanks @barrust. PR in #169 for consideration.

**Comment by barrust** [comment]
Thank you! I will look at it shortly. I actually started a new branch locally to add country names in English, Spanish, French, and Italian along with rebuilding the dictionaries. 

I might use the one I am working on as it includes splitting the words, etc. built into the `build_dictionary.py` script.

**Comment by barrust** [comment]
published [version 0.8.1](https://github.com/barrust/pyspellchecker/releases/tag/v0.8.1) with the updated dictionaries
